### DISCUSSION TITLE
Adding QGIS under supported STAC tools

### DISCUSSION THREAD

**Discussion by saberraz** [stac-utils/.github#8] [General]
With the recent work in QGIS core, would it be possible to add the software under https://github.com/stac-utils#other? You no longer need the plugin to connect to STAC servers and the functionality comes out of box with the main QGIS application.

**Comment by gadomski** [comment]
Very cool! Excited to see out-of-the-box support. Should we add a note to https://github.com/stac-utils/qgis-stac-plugin so folks don't get confused on whether they need it? Or does that plugin still provide additional functionality?

EDIT: https://github.com/stac-utils/.github/issues/9

**Reply by saberraz** [reply]
We have been in contact with the plugin developers and they have provided great inspiration and insight. The plugin can be used for earlier version of QGIS without STAC support. So, best to have it as an option for the time being and then eventually make it redundant (if developers agree) once all the functionalities are available in core.
